subprojects/android-test/test-report/src/main/kotlin/com/avito/android/test/report/video/VideoCapturer.kt

```diff
@@ -61,13 +64,11 @@ class VideoCapturerImpl(
         }
     }
 
-    @SuppressLint("LogNotTimber")
     @Synchronized
-    override fun stop(): VideoCapturer.RecordResult {
-        val state = state
-        val result = when {
-            state is State.Recording -> {
-                val (videoFile, outputFile) = state
+    override fun stop(): VideoCapturer.RecordResult =
+        when (val castHelperLocalState = state) {
```

Comment:
Here is no casting

Reply:
`Smart cast to com.avito.android.test.report.video.VideoCapturerImpl.State.Recording`